sql/auto_install.sql

```diff
@@ -1,27 +1,94 @@
+-- | with this program; if not, contact CiviCRM LLC                     |
+-- | at info[AT]civicrm[DOT]org. If you have questions about the        |
+-- | GNU Affero General Public License or the licensing of CiviCRM,     |
+-- | see the CiviCRM license FAQ at http://civicrm.org/licensing        |
+-- +--------------------------------------------------------------------+
+--
+-- Generated from schema.tpl
+-- DO NOT EDIT.  Generated by CRM_Core_CodeGen
+--
 
-SET FOREIGN_KEY_CHECKS=1;
-CREATE TABLE `membershipextras_contribrecur_lineitem` (
-  `id` int unsigned NOT NULL AUTO_INCREMENT COMMENT 'Discount Item ID',
-  `contribution_recur_id` int unsigned NOT NULL COMMENT 'ID of the recurring contribution.',
-  `line_item_id` int unsigned NOT NULL COMMENT 'ID of the line item related to the recurring contribution.',
-  `start_date` datetime NULL COMMENT 'Start date of the period for the membership/recurring contribution.',
-  `end_date` datetime NULL COMMENT 'End date of the period for the membership/recurring contribution.',
-  `auto_renew` datetime NULL COMMENT 'If the line-item should be auto-renewed or not.',
 
-  PRIMARY KEY (`id`),
+-- +--------------------------------------------------------------------+
+-- | CiviCRM version 4.7                                                |
+-- +--------------------------------------------------------------------+
+-- | Copyright CiviCRM LLC (c) 2004-2018                                |
+-- +--------------------------------------------------------------------+
+-- | This file is a part of CiviCRM.                                    |
+-- |                                                                    |
+-- | CiviCRM is free software; you can copy, modify, and distribute it  |
+-- | under the terms of the GNU Affero General Public License           |
+-- | Version 3, 19 November 2007 and the CiviCRM Licensing Exception.   |
+-- |                                                                    |
+-- | CiviCRM is distributed in the hope that it will be useful, but     |
+-- | WITHOUT ANY WARRANTY; without even the implied warranty of         |
+-- | MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.               |
+-- | See the GNU Affero General Public License for more details.        |
+-- |                                                                    |
+-- | You should have received a copy of the GNU Affero General Public   |
+-- | License and the CiviCRM Licensing Exception along                  |
+-- | with this program; if not, contact CiviCRM LLC                     |
+-- | at info[AT]civicrm[DOT]org. If you have questions about the        |
+-- | GNU Affero General Public License or the licensing of CiviCRM,     |
+-- | see the CiviCRM license FAQ at http://civicrm.org/licensing        |
+-- +--------------------------------------------------------------------+
+--
+-- Generated from drop.tpl
+-- DO NOT EDIT.  Generated by CRM_Core_CodeGen
```

Comment:
All these comments should also be deleted. Seems they're actually duplicated.

Reply:
Done